realtime/websocket/__init__.py

```diff
@@ -18,13 +18,10 @@ def websocket(path: str = "/"):
     TODO: Add video support
     """
 
-    def decorator(func):
-        async def websocket_handler(websocket: WebSocket):
-            RealtimeServer().add_connection()
+    def decorator(func: Callable) -> Callable:
+        @functools.wraps(func)
+        async def wrapper(*args, **kwargs) -> None:
             try:
```

Comment:
You don't need a wrapper. The wrapper will never be called and your code will not run.